### ISSUE TITLE
Several concerns of this paper

### ISSUE THREAD

**Issue by neuripsanonymous** [kuandeng/LightGCN#4]
First, I have noticed that this paper had been submitted to arxiv with author information before it is submitted to SIGIR 2020. There seems to be no significant difference between the two versions. So I think it is a minor violation of the Blind Review Process. 

Second, the general idea of this paper is very similar to the AAAI 20 paper "Revisiting Graph based Collaborative Filtering: A Linear Residual Graph Convolutional Network Approach". I used to think it might be because two independent organizations came up with similar ideas, until I noticed that the authors of the two papers overlapped (e.g., Meng, Wang). People who have concerns should take a look at the above two papers. 

Looking forward to a response.

**Comment by hexiangnan** [comment]
Thanks for pointing that out.

1. We put the paper in arxiv after it is submitted to SIGIR. The SIGIR
2020 webpage (http://sigir.org/sigir2020/call-for-full-papers/) indicates
that "While authors can upload to institutional or other preprint
repositories such as arXiv.org before reviewing is complete, we generally
discourage this since it places anonymity at risk (which could result in a
negative outcome of the reviewing process).". So we changed the title a
little bit when uploading it to arxiv. It does not volidate the
double-blind policy of SIGIR.

2. The AAAI 2020 work was carried at the same time (i.e., June to Sep 2019)
but the main authors do not know each other. LightGCN was first submitted
to WWW (the same submission time with AAAI) but got rejected. Although the
shared author Wang Meng participated in the discussion of the two papers,
he thought the idea, the final approach, and how to interpret the approach
are very different. The main differences are: 1) LightGCN removes both
nonlinearity and feature transformation and justify the rationality,
whereas RGCN only removes nonlinearity; 2) LightGCN further removes
self-connection in the adjacency matrix, since it proves in theory that the
addition-based combination layer subsumes the self-connections, whereas
RGCN still keeps self-connections since it differently follow traditional
GCN; 3) LightGCN sums all layers as the final representation, which can be
interpreted from the perspective of self-connection and saves the embedding
size efficiently, whereas RGCN still concatenates all layers (same as NGCF)
and interprets it as a residual network structure.

In short, although the motivation shares a bit similarity (training
difficulty of GCN for recommendation), the difference between LightGCN and
RGCN are significant: in the idea, the approach, the way to interpret the
approach, and the experiments. Thus, we believe that LightGCN is worth as
an independent research paper (moreover, LightGCN performs more rigorous
experiments on the exact same setting of NGCF). Since RGCN was not
publicized before the SIGIR deadline (neither in AAAI proceeding nor in
arxiv), we cannot refer to the RGCN paper when submitting LightGCN to
SIGIR. So after LightGCN is accepted, we add citation of RGCN and
discuss the difference with it in related work in the camera-ready version.

Hope it clarifies your concerns.

Cheers,
Xiangnan

**Comment by neuripsanonymous** [comment]
Thanks for your reply!

For the first concern, although the title is changed a little, it can still be clearly identified as the same paper. Since the call-for-paper didn't prohibit this kind of behavior, I can only say that this kind of behavior is not worth encouraging.

For the second concern, I can understand your response if the authors of the two papers are not overlapped. But you have the same authors, so it is a bit strange for me that you don't know each other's work. Moreover, I still think that the difference between LightGCN and LR-GCCF is actually not significant. Because the main contribution and "spotlight" of the two papers are the same: simplifying the design of GCN to make it more concise and appropriate for recommendation. The difference between these two papers are just little "tricks" but not important.  

In short, I understand, but I don't think this is good for the research community.

**Comment by kuandeng** [comment]
Hi, I think you are a Chinese and some things are hard to express in English. Can you give me your phone number so that I can explain the process to you? No worry, I just want to have a pure academic discussion with you and I donot have any other meaning.
